analyzers/src/SonarAnalyzer.Common/Helpers/TypeHelper.cs

```diff
@@ -43,6 +43,12 @@ internal static class TypeHelper
         public static bool Is(this ITypeSymbol self, TypeKind typeKind) =>
             self is { } && self.TypeKind == typeKind;
 
+        public static bool IsNullable(this ITypeSymbol self) =>
```

Comment:
This might be confused with nullable reference types. I would be more explicit here like you were before with `IsNullableValueType`